doc/references/users.rst

```diff
@@ -94,6 +94,8 @@ Companies, Governmental and Non-Governmental Organisations
 
 * `TransnetBW <https://www.transnetbw.de/>`_ (the electricity transmission system operator in southwest Germany), `ONTRAS <https://www.ontras.com/>`_, (the gas transmission system operator in eastern Germany) and `d-fine <https://www.d-fine.com/>`_ (the consultancy firm): used PyPSA-Eur-Sec for a study in 2020 of the grid requirements in 2050 with a 90% reduction of carbon dioxide emissions in electricity, buildings and transport, see `Stromnetz 2050 <https://www.transnetbw.de/de/stromnetz2050/>`_, for an article in a 2021 issue of Energiewirtschaftliche Tagesfragen `Die Rolle von Wasserstoff in einem klimaneutralen europäischen Energiesystem – eine modellbasierte Analyse bis 2050 <https://www.d-fine.com/fileadmin/user_upload/Wasserstoff_et_0102-2021.pdf>`_, and for their 2022 study `Energy System 2050 - Towards a decarbonised Europe <https://www.energysystem2050.net/>`_.
 
+* `ENTSO-E <https://www.entsoe.eu/>`_ (the Euroean electricity transmission system operator assocciation) launched in collaboration with `Open Energy Transition <https://openenergytransition.org/>`_ a research and innovation project to develop a complementary open-source tool in PyPSA for `Scenario Building and Cost-Benefit Analysis within the TYNDP process <https://www.linkedin.com/posts/entso-e_energytransition-opensource-innovation-activity-7293296246813851649-2ynL>`_.
```

Comment:
@gincrement again a typo. `Euroean -> European`

Maybe use the official abbreviation in the brackets -> (European Network of Transmission System Operators for Electricity)